triggers/tests/BUILD.bazel

```diff
@@ -103,77 +118,24 @@ dependencies:
   - daml-script.dar
 EOF
       test -z "{lf_version}" || echo "build-options: [--target={lf_version}]" >> $$TMP_DIR/daml.yaml
-      $(location //compiler/damlc) build --project-root=$$TMP_DIR --ghc-option=-Werror -o $$PWD/$(location acs{suffix}.dar)
+      $(location //compiler/damlc) build --project-root=$$TMP_DIR --ghc-option=-Werror -o $$PWD/$(location acs{version_suffix}.dar)
       rm -rf $$TMP_DIR
     """.format(
                 lf_version = lf_version,
                 sdk = sdk_version,
-                suffix = suffix,
+                version_suffix = version_suffix,
             ),
             tools = ["//compiler/damlc"],
             visibility = ["//visibility:public"],
         ),
         da_scala_test_suite(
-            name = "trigger-integration-tests" + suffix,
-            srcs = [
-                "src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/%s.scala" % f
-                for f in [
```

Comment:
I see that here we explicitly enumerate those files we wanted to test, in three locations:

`test-utils` has
```
        "src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala",
        "src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractTriggerTest.scala",
        "src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/CompiledDar.scala",
```

`trigger-integration-tests` has 
```
                    "FuncTestsStaticTime",
                    "FuncTestsWallClock",
                    "Jwt",
                    "Tls",
                    "RunnerSpec",
                    "UnfoldStateSpec",
                    "ConfigSpec",
                    "InterfaceSpec",
```
(now split up into two groups RIGGER_INTEGRATION_TESTS and TRIGGER_CANTON_INTEGRATION_TESTS)

And `trigger-failure-integration-tests` has `src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/LoadTesting.scala`.

Would it be worthwhile to split those three (now four) groups we're testing into four separate subfolders and use filegroups to pick them up instead? That way the file structure documents that these tests are interacted with differently, and when you add a new test file, you don't need to remember to add it here in BUILD.bazel.

Feel free to disregard is this represents and annoying amount of extra work though - aside from this idea, PR looks great.